vllm/v1/executor/abstract.py

```diff
@@ -1,63 +1,75 @@
-from abc import ABC, abstractmethod
 from typing import Type
 
 from vllm.config import VllmConfig
+from vllm.executor.executor_base import ExecutorBase
+from vllm.executor.ray_distributed_executor import RayDistributedExecutor
+from vllm.executor.uniproc_executor import (ExecutorWithExternalLauncher,
+                                            UniProcExecutor)
 from vllm.v1.kv_cache_interface import KVCacheConfig, KVCacheSpec
 from vllm.v1.outputs import ModelRunnerOutput
 
 
-class Executor(ABC):
-    """Abstract class for executors."""
+class Executor(ExecutorBase):
+    """Abstract class for v1 executors, mainly define some new methods."""
```

Comment:
A more clear comment: "Abstract class for v1 executors, mainly define some methods for v1. Define methods shared by v0 and v1 in ExecutorBase" And how to add an interface to only v0?

Reply:
fixed in [dc2a886](https://github.com/vllm-project/vllm/pull/12171/commits/dc2a8866688a50bfa1eb96831fce58bcba6270db)

Reply:
> how to add an interface to only v0

we don't introduce new features that only work for v0.